### DISCUSSION TITLE
How to disable underline in button link?

### DISCUSSION THREAD

**Discussion by tony9527167** [mantinedev/mantine#6398] [Q&A]
This snippet does not work

```
        <Button
          component="a"
          underline="never"
          href={configs.URL_OPEN_AFTER_INSTALL}
          target="_blank"
          leftSection={<IconBook size={14} />}
          variant="subtle"
        >button link</Button>
```

![image](https://github.com/mantinedev/mantine/assets/63030915/cabd3604-45a0-4a05-8e30-e447f543e373)

```
  System:
    OS: Windows 11 10.0.22631
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
    Memory: 4.45 GB / 15.67 GB
  Binaries:
    Node: 21.7.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.22 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 10.5.0 - C:\Program Files\nodejs\npm.CMD
    pnpm: 9.3.0 - ~\AppData\Local\pnpm\pnpm.CMD
    bun: 1.1.2 - ~\.bun\bin\bun.EXE
  Browsers:
    Edge: Chromium (126.0.2592.56)
    Internet Explorer: 11.0.22621.3527
  npmPackages:
    @mantine/core: ^7.10.1 => 7.10.1 

```

**Comment by rtivital** [comment]
Button component does not have underline styles, there is nothing to disable.

**Reply by tony9527167** [reply]
v6 supports pass underline=false to `<Button component={"a"} underline={false} />` or its default no underline.
please add support for passing underline(or any attributes for a), or make it `underline:none` by default.

It's common cases build button clickable anchor.

**Reply by rtivital** [reply]
Please provide a sandbox with a minimal reproduction where the button has underline

**Comment by sirianni** [comment]
The issue is if the `<Button>` is nested within a `<Typography>` element.  I think many applications still put `<Typography>` near the root of the component tree.  Is this no longer a best practice in the latest mantine version?  It's not clear from the docs.
